projects/rero/ng-core/src/lib/record/search/result/record-search-result.component.spec.ts

```diff
@@ -38,6 +39,7 @@ describe('RecordSearchResultComponent', () => {
         RecordSearchResultComponent
       ],
       imports: [
+        RouterTestingModule,
```

Comment:
Why you have to do this ?

Reply:
Because the tests fail without this import.